<!DOCTYPE HTML>
<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">

  <title>Yutong Zhu</title>
  
  <meta name="author" content="Yutong Zhu">
  <meta name="keywords" content="Yutong Zhu; Irina Zhu; Yutong (Irina) Zhu; University of Pennsylvania; Bioengineering; Biomedical Engineering; Johns Hopkins University; Johns Hopkins School of Medicine; Chemical and Biomolecular Engineering; Computational Biology; Spatial Biology; Digital Pathology; AI in Medicine; Tissue Engineering; Regenerative Medicine; Univeristy of Toronto; Engineering Science; U of T EngSci">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="google-site-verification" content="YtwRo3klD4s5aXhFIGHG3DhISojFN8Tmur_3dbOGh9c" />
  
  <link rel="stylesheet" type="text/css" href="stylesheet.css">
  <link rel="icon" type="image/png" href="images/dna.png">
</head>

<body>
  <div class ="background "id="vanta"></div>
  <script src="three.r119.min.js"></script>
  <script src="vanta.net.min.js"></script>
  <script>
  VANTA.NET({
    el: "#vanta",
    mouseControls: true,
    touchControls: true,
    gyroControls: false,
    minHeight: 200.00,
    minWidth: 200.00,
    scale: 1.00,
    scaleMobile: 1.00,
    color: 0x9b04bb,
    backgroundColor: 0x211d31
})
</script>
  <table style="width:100%;max-width:800px;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;background-color:#f1f1f1"><tbody>
    <tr style="padding:0px">
      <td style="padding:0px">
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr style="padding:0px">
            <td style="padding:2.5%;width:63%;vertical-align:middle">
              <p style="text-align:center">
                <name>Yutong (Irina) Zhu</name>
              </p>
              <p> Hi there! I'm glad you made it to my digital portfolio :)
              </p>
              <p> I am a 2nd-year <strong> Bioengineering PhD</strong> student at <strong>University of Pennsylvania</strong>, advised by <a href="https://www.theoldridgelab.com/">Derek Oldridge</a>. 
              </p>
              <p> I am interested in developing <strong> statistical and AI-driven analytical tools </strong> that integrate spatial-omic technologies with conventional pathology imaging to study spatial biology. My goal is to enhance our understanding of the <strong> molecular, cellular, and architectural foundations of human health and disease in immunological context </strong>.
              </p>
              <p style="text-align:center">
                <a href="mailto:irinaz@seas.upenn.edu">Email</a> &nbsp/&nbsp
                <a href="data/Yutong_Zhu_CV.pdf">CV</a> &nbsp/&nbsp
                <a href="https://www.linkedin.com/in/yutong-irina-zhu/">LinkedIn</a> &nbsp/&nbsp
                <a href="https://github.com/yutong-zhu">GitHub</a> &nbsp/&nbsp
                <a href="https://scholar.google.com/citations?user=KVa3ycEAAAAJ&hl=en">Google Scholar</a> 
              </p>
            </td>
            <td style="padding:2.5%;width:40%;max-width:40%">
              <a href="images/Irina_square_new.jpg"><img style="width:100%;max-width:100%" alt="profile photo" src="images/Irina_circle_new.png" class="hoverZoomLink"></a>
            </td>
          </tr>
        </tbody></table>

        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr>
            <td style="padding:20px;width:100%;vertical-align:middle">
              <heading>Education</heading>
            </td>
          </tr>

        </tbody></table>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src='images/UPenn.png' width="160">
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <papertitle>P.h.D. in Bioengineering</papertitle>
              <br>
              School of Engineering and Applied Science, University of Pennsylvania
              <br>
              Children's Hospital of Philadelphia
              <br>
              Sep 2024 - Present | Philadelphia, PA, USA
              <br>
              Advisor: <a href="https://scholar-google-com.proxy.library.upenn.edu/citations?user=upzVk4wAAAAJ&hl=zh-CN">Derek Oldridge</a>
            </td>
          </tr>         

        </tbody></table>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src='images/JHMI.jpg' width="160">
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <papertitle>M.S.E. in Chemical and Biomolecular Engineering</papertitle>
              <br>
              Whiting School of Engineering, Johns Hopkins University
              <br>
              Sep 2022 - Apr 2024 | Baltimore, MD, USA
              <br>
              <br>
              <em>GPA: 4.0/4.0</em>
              <br> 
              Advisor: <a href="https://scholar.google.com/citations?user=UJP_JHEAAAAJ&hl=en">Denis Wirtz</a> & <a href="https://scholar.google.com/citations?hl=en&user=l0IwtH8AAAAJ">Ashley Kiemen</a>
            </td>
          </tr>

        </tbody></table>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src='images/UofT.png' width="160">
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <papertitle>B.A.Sc. in Engineering Science, Biomedical Systems Engineering</papertitle>
              <br>
              Faculty of Applied Science and Engineering, University of Toronto
              <br>
              Sep 2017 - Apr 2022 | Toronto, ON, Canada
              <br>
              <br>
              <em> Dean’s Honors List, 2020-2022 </em>
              <br>
              <em> U of T Entrance Scholarship, 2017 </em>
              <br> 
              <br>
              Thesis Advisor: <a href="https://scholar-google-com.proxy.library.upenn.edu/citations?user=8PaPFSUAAAAJ&hl=en">Eli Sone</a> (2021-2022)
              <br>
              Research Advisor: <a href="https://scholar-google-com.proxy.library.upenn.edu/citations?user=3srbJBIAAAAJ&hl=en">Michael Garton</a> (2021)
              <br>
              Research Advisor: <a href="https://scholar-google-com.proxy.library.upenn.edu/citations?user=gzuE2kQAAAAJ&hl=en">Alison McGuigan</a> (2019)
            </td>
          </tr>
        </tbody></table>

        
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr>
            <td style="padding:20px;width:100%;vertical-align:middle">
              <heading>Publications</heading>
            </td>
          </tr>


      </tbody></table>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/gng4.png" width="160">
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <papertitle>Multimodal analysis defines GNG4 as a distinguishing feature of germinal center-positioned CD4 T follicular helper cells in humans</papertitle>
              <br>
              Sam Barnett Dubensky, 
              <strong>Yutong Zhu</strong>, 
              Molly Gallagher, 
              Kingsley Gideon Kumashie, 
              Tianyu Lu, 
              Jonathan Tedesco, 
              Nina De Luna, 
              Katherine Premo, 
              Yi Qi, 
              Suzanna Rachimi, 
              Emylette Cruz Cabrera, 
              Bria Fulmer, 
              Ijeoma C. Meremikwu, 
              Ashley Carter,
              Sarah E. Henrickson,
              Neil Romberg, 
              Amy E. Baxter, 
              Derek A. Oldridge,
              Laura A. Vella
              <br>
              <br>
              <em> <strong>Submitted, 2025</strong> </em>
              <br>
              <a href="https://urldefense.com/v3/__https://doi.org/10.64898/2025.12.10.693235__;!!IBzWLUs!XKC7FellE7FU3BYw5VpFnpZWoMLCGRMh8aJfp1oC5gyMq8R7XyASXCvt81RVTvCwAMX3ik4-88j7FWts5g7kAmMsXUQS$"><em>bioRxiv preprint</em> </a>
              <br>
              <p>
              Using multimodal single-cell and spatial transcriptomic profiling of human tonsils and blood, we define distinct CD4 T follicular helper (Tfh) cell states and identify GNG4 as a hallmark of activated, germinal center–associated Tfh cells. GNG4 expression more specifically marks GC-localized Tfh than canonical markers, distinguishing activated GC Tfh from resting, non-GC, Th17-polarized Tfh states.
              </p>

            </td>
          </tr> 
      

      </tbody></table>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/USCAP_figure.jpg" width="160">
              <img src="images/USCAP_figure_2.jpg" width="160">
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <papertitle>Artificial Intelligence Based-Prediction of Recurrence of Surgically Resected Pancreatic Neuroendocrine Tumors in a Multi-Institutional Cohort Reveals Histological Patterns Associated with Risk</papertitle>
              <br>
              Arrun Sivasubramanian,
              Lucie S R Dequiedt, 
              Yu Shen, 
              Valentina Matos-Romero, 
              <strong>Yutong Zhu</strong>, 
              Uwakmfon-Abasi Imoh Ebong, 
              Zoe Gaillard,
              Ian Reucroft,
              Chen Mayer, 
              Hyun-Nam Yun, 
              Seung-Mo Hong, 
              Jinho Shin,
              Atsuko Kasajima, 
              Günter Klöppel, 
              Jin He, 
              Amanda L Blackford, 
              Ralph H. Hruban, 
              Ashley L Kiemen
              <br>
              <br>
              <em> <strong> USCAP Annual Meeting (Accepted), 2025</strong> </em>
              <br>
              <a href="https://uscap.org/uscap-annual-meeting/"><em></em> Website</a>
              <br>
              <p>
                We developed an AI-based histopathology model to distinguish long-term disease-free PanNET patients from those who later recurred, using 3,888 H&E images from 926 resections. The pipeline achieved high accuracy (87–89%) across recurrence thresholds and attention maps revealed that small regions, often at invasive edges or vascular invasion—drove prediction. These findings highlight AI as a promising tool for postoperative risk stratification in PanNETs.
              </p>

            </td>
          </tr> 

      </tbody></table>
      <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
        <tr>
          <td style="padding:20px;width:25%;vertical-align:middle">
             <video width="160" height="140" autoplay loop muted>
              <source src="images/3Dpanin1.mp4" type="video/mp4" />
             </video>
             <video width="160" height="140" autoplay loop muted>
              <source src="images/3Dpanin.mp4" type="video/mp4" />
             </video>
          </td>
          <td style="padding:20px;width:75%;vertical-align:middle">
            <papertitle>3D genomic mapping reveals multifocality of human pancreatic precancers</papertitle>
            <br>
            Alicia M. Braxton,
            Ashley L. Kiemen*,
            Mia P. Grahn,
            André Forjaz,
            Jeeun Parksong,
            Jaanvi Mahesh Babu,
            Jiaying Lai,
            lily zheng,
            Noushin Niknafs,
            Liping Jiang,
            Haixia Cheng,
            Qianqian Song,
            Rebecca Reichel,
            Sarah Graham,
            Alexander I. Damanakis,
            Catherine G Fischer,
            Stephanie Mou,
            Cameron Metz,
            Julie Granger,
            Xiao-Ding Liu,
            Niklas Bachmann,
            <strong>Yutong Zhu</strong>,
            YunZhou Liu,
            Cristina Almagro-Perez,
            Ann Chenyu Jiang,
            Jeonghyun Yoo,
            Bridgette Kim,
            Scott Du,
            Eli Foster,
            Jocelyn Y. Hsu,
            Paula Andreu Rivera,
            Linda C. Chu,
            Fengze Liu,
            Elliot K. Fishman,
            Alan Yuille,
            Nicholas J. Roberts,
            Elizabeth D. Thompson,
            Robert B. Scharpf,
            Toby C. Cornish,
            Yuchen Jiao,
            Rachel Karchin,
            Ralph H. Hruban,
            Pei-Hsun Wu,
            Denis Wirtz*,
            Laura D. Wood*
            <br>
            <br>
            <em> <strong> Nature, 2024 </strong> </em>
            <br>
              <a href="https://www.nature.com/articles/s41586-024-07359-3"><em> Article </em> </a>
              <br>
            <p>
              The study involves quantitative assessment of human pancreatic intraepithelial neoplasia (PanIN) using CODA, a novel machine-learning pipeline for 3D image analysis that generates quantifiable models of large pieces of human pancreas with single-cell resolution. By leveraging CODA, for the first time, we are able to describe the spatial and genetic multifocality of human PanINs, providing important insights into the initiation and progression of pancreatic neoplasia.
            </p>
          </td>
            </td>
          </tr>

        </tbody></table>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/panin1.png" width="160">
              <img src="images/panin2.png" width="160">
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <papertitle>PanIN or IPMN? Redefining Lesion Size in 3 Dimensions</papertitle>
              <br>
            Ashley L. Kiemen,
            Lucie Dequiedt,
            Yu Shen,
            <strong>Yutong Zhu</strong>,
            Valentina Matos-Romero,
            André Forjaz,
            Kurtis Campbell,
            Will Dhana,
            Toby C. Cornish,
            Alicia M. Braxton,
            Pei-Hsun Wu,
            Elliot K. Fishman,
            Laura D. Wood
            Denis Wirtz,
            Ralph H. Hruban
            <br>
            <br>
            <em> <strong> American Journal of Surgical Pathology, 2024</strong> </em>
            <br>
              <a href="https://journals.lww.com/ajsp/abstract/2024/07000/panin_or_ipmn__redefining_lesion_size_in_3.8.aspx"><em> Article </em> </a>
              <br>
              <p>
                This study reevaluates the conventional clinical classification of precursor lesions in Pancreatic Ductal Adenocarcinoma (PDAC), namely PanIN (Pancreatic Intraepithelial Neoplasia) and IPMN (Intraductal Papillary Mucinous Neoplasms). It reveals the inadequacy of the traditional classification when assessing high-resolution 3D images. By constructing cellular-resolution 3D maps, the research uncovers instances where some PanINs grow to macroscopic sizes in narrow ducts, meeting the criteria for IPMNs.
              </p>
            </td>
          </td>
        </tr>


      </tbody></table>
      <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
        <tr>
          <td style="padding:20px;width:25%;vertical-align:middle">
            <img src="images/mcguigan.png" width="160">
          </td>
          <td style="padding:20px;width:75%;vertical-align:middle">
            <papertitle>An off-the-shelf multi-well scaffold-supported platform for tumour organoid-based tissues</papertitle>
            <br>
            Nancy T. Li,
            Nila C. Wu,
            Ruonan Cao,
            Jose L. Cadavid,
            Simon Latour,
            <strong>Yutong Zhu</strong>,
            Mirjana Mijalkovic,
            Reza Roozitalab,
            Natalie Landon-Brace,
            Faiyaz Notta,
            Alison P. McGuigan
            <br>
            <br>
            <em> <strong>Biomaterials, 2022</strong> </em>
            <br>
            <a href="https://www.sciencedirect.com/science/article/pii/S0142961222005233"> <em>Article</em></a>
            <br>
            <p>
              The study focuses on developing a Scaffold-supported Platform for Organoid-based Tissues (SPOT) platform compatible with generic 96/384 platforms, using 3D design and bioprinting techniques. SPOT enables a 3D heterogeneous environment that mimics the tumor microenvironment, serving as a useful tool to assess the drug sensitivity of patient-derived organoids and can be easily integrated into the drug discovery pipeline. 
          </td>
        </tr>  
    


      <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
        <tr>
          <td style="padding:20px;width:100%;vertical-align:middle">
            <heading>Past Research & Presentations</heading>
          </td>
        </tr>

      </tbody></table>
    <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
      <tr>
        <td style="padding:20px;width:25%;vertical-align:middle">
          <video width="160" height="120" autoplay loop muted>
            <source src="images/PM2.mp4" type="video/mp4" />
           </video>
           <video width="160" height="120" autoplay loop muted>
            <source src="images/OM1.mp4" type="video/mp4" />
           </video>
        </td>
        <td style="padding:20px;width:75%;vertical-align:middle">
          <papertitle>Three-dimensional reconstruction and multi-organ mapping of the female mouse reproductive system as a function of age</papertitle>
          <br>
          <strong>Yutong Zhu</strong>,
          Mia P. Grahn,
          Aanya Kheterpal,
          Ashleigh Crawford,
          Denis Wirtz,
          Ashley L. Kiemen
          <br>
          <br>
          <em> Johns Hopkins Annual ChemBE Research Symposium (Star poster), 2023 </em>
          <br>
          <em>Presenter: <strong>Yutong Zhu</strong></em>
          <br>
          <a href="data/gyn_poster.pdf"> Poster</a>
          <br>
          <p>
            The study involves the construction of 3D, multi-organ volumes of the normal reproductive system of mice at different age groups using CODA, a novel machine-learning pipeline for 3D image analysis, to qualitatively and quantitatively identify major changes to ovary, fallopian, cervix, and vagina morphology and immune infiltration with age.
          </p>
        </td>
      </td>
    </tr>


      </tbody></table>
      <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
        <tr>
          <td style="padding:20px;width:25%;vertical-align:middle">
            <img src="images/mussel.png" width="160">
          </td>
          <td style="padding:20px;width:75%;vertical-align:middle">
            <papertitle>Characterization of post-translational modification (PTM) of freshwater quagga mussel adhesive protein</papertitle>
            <br>
            <strong>Yutong Zhu</strong>,
            Angelico Obille,
            Eli Sone
            <br>
            <br>
            <em> Engineering Science Undergraduate Thesis Presentation, 2022 </em>
            <br>
            <em>Presenter: <strong>Yutong Zhu</strong> </em>
            <br>
            <a href="data/Thesisslide.pdf">Thesis Defense</a> /
            <a href="data/Thesis.pdf">Thesis Report</a> 
            <br>
            <p>
              The study focuses on the distinct adhesion mechanism employed by freshwater mussels. By quantitatively analyzing localization profiles and post-translational modifications of the adhesive proteins using a staining-based approach, this work provides a well-established role model for developing novel bioadhesives such as self-healing materials and advanced coatings in aqueous environments.
            </p>
          </td>
        </tr>  

        </tbody></table>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/garton.png" width="160">
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <papertitle>Design of landing pad cell lines targeting specific safe harbour sites for efficient gene editing</papertitle>
              <br>
              Professional Co-op Experience Year,  <a href="http://www.gartonlab.org/">Garton Lab</a>, University of Toronto
              <br>
              <strong>Yutong Zhu</strong>,
              Aaron Rosenstein,
              Michael Garton
              <br>
              <a href="data/landingpad.pdf">Final Presentation Slides</a>
              <br>
              <p>
                Designed and developed a library of landing pad cell lines targeting pSH231 and AAVS1 safe harbor sites. The library provides an efficient approach for recombinase-mediated cassette exchange with a specific promoter and gene of interest. 
              </p>
              <em>*Any future work involving the utilization of the landing pad cell lines will credit this work done by <strong>Yutong Zhu</strong>, Aaron Rosenstein and Michael Garton</em>
            </td>
          </tr> 
        
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr>
            <td style="padding:20px;width:100%;vertical-align:middle">
              <heading>Engineering Projects</heading>
            </td>
          </tr>
        </tbody></table>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/Capstone.png" width="160">
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <papertitle>Subtalar Joint Preparation Device for Fusion During Tibiotalocalcaneal Arthrodesis</papertitle>
              <br>
              BME489 Biomedical Systems Engineering Design, University of Toronto
              <br>
              <a href="data/capstone_final_report.pdf">Final Report</a> /
              <a href="data/capstone_final_presentation.pdf">Final Presentation Slides</a>
              <br>
              <p>
                Designed and simulated a device that prepares the subtalar joint for fusion during tibiotalocaneal arthrodesis, which consists of 2 modular components: a cutting tool and a guide tube.
              </p>
            </td>
          </tr>  

        </tbody></table>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/DesignProject.png" width="160">
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <papertitle>Effects of Drugs on Cancer Cell Metastasis</papertitle>
              <br>
              BME346 Biomedical Engineering & Omics Technologies, University of Toronto
              <br>
              <a href="data/design_project_final_presentation.pdf">Final Presentation</a> /
              <a href="data/design_project_proposal.pdf">Project Proposal</a>
              <br>
              <p>
              Discovered great potential in the combination of platelet inhibition and natural killer (NK) cell enhancement, where platelet inhibition sensitizes tumor cells to NK cell activity and NK enhancement increases cytotoxic effects.
              </p>
            </td>
          </tr>  

        </tbody></table>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/utat.jpeg" width="160">
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://www.utat.ca/space-systems">
              <papertitle>“HERON” Biological Payload Design</papertitle>
              </a>
              <br>
              <a href="https://www.utat.ca/">University of Toronto Aerospace Team</a>
              <br>
              <a href="data/payload-assembly-full-v2.mp4">Demo</a>
              <br>
              <p>
                Engineered <em>C. albicans</em> to express GFP with specific genes and developed a statistical method for quantifying this gene expression following long-time space mission.
              </p>
            </td>
          </tr>  

        </tbody></table>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/AER201.jpeg" width="160">
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://ryan-zeyuan-chen.github.io/AER201/">
              <papertitle>Traffic Cone Deployment Robot</papertitle>
              </a>
              <br>
              <a href="http://aer201.aerospace.utoronto.ca/">AER201 Engineering Design</a>, University of Toronto
              <br>
              <a href="data/AER201.pdf">Final Report</a> /
              <a href="data/AER201.mp4">Demo</a>
              <br>
              <p>
                Designed and fabricated of a fully autonomous mobile robot that can deploy miniature traffic cones based on the detection of "cracks" and "holes".  The robot was constructed with omnidirectional mecanum wheels, infrared sensors, as well as an interactive LCD and keyboard user interface.
              </p>
            </td>
          </tr>  
        </tbody></table>
      </td>
    </tr>
  </table>
  <div align="center" style="margin:auto;padding-top:10px">
    <div style="width:10%">
      <script type="text/javascript" id="clstr_globe" src="//clustrmaps.com/globe.js?d=iLsqH5SOyzwcKeT8gK8MaOwAJBw9fhvnGU60rk8l-Oo"></script>
    </div>
  </div>
</body>

</html>
